azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/state/LocalUserState.kt

```diff
@@ -78,4 +79,5 @@ internal data class LocalUserState(
         false,
         false
     ),
+    val localParticipantRole: CallCompositeInternalParticipantRole?
```

Comment:
Should we use CallCompositeInternalParticipantRole.UNINITIALIZED as a default value instead of null?

Reply:
I think null will be good Pavel, native sdk set to UNINITIALIZED